### PR TITLE
Export unmodified TA blueprints.

### PR DESCRIPTION
**Pull request by PhilPJL**
Changed definition cache to keep an original copy of TA blueprints.
During export the original copy is exported in place of the current blueprint which may have been modified by CE.
Updated TA and CE blueprints.

TODO: export TA blueprints that were modified by CE.

### CODE REVIEW

File: SolastaCommunityExpansion/Diagnostics/CommunityExpansionBlueprints/ItemDefinition/Enchanted_Quarterstaff_Acuteness.json [hunk 1]
```diff
@@ -378,24 +379,18 @@
...
   "isContainerItem": false,
   "isLightSourceItem": false,
-  "isFocusItem": true,
-  "focusItemDefinition": {
-    "$id": "35",
-    "$type": "FocusItemDescription, Assembly-CSharp",
-    "focusType": "Arcane",
-    "shownAsFocus": true
-  },
+  "isFocusItem": false,
```
Comment (ChrisPJohn) [inline SolastaCommunityExpansion/Diagnostics/CommunityExpansionBlueprints/ItemDefinition/Enchanted_Quarterstaff_Acuteness.json:382]: There's a bunch of  quarterstaffs that changed from arcane focus to not a focus item. I think there's a CE setting to make all quarterstaffs an arcane focus, was one export done with that turned on?

File: SolastaCommunityExpansion/Diagnostics/OfficialBlueprints/ConditionDefinition/ConditionTrueStrike.json [hunk 2]
```diff
@@ -250,8 +250,8 @@
     "$id": "30",
     "$type": "GuiPresentation, Assembly-CSharp",
     "hidden": false,
-    "title": "Condition/&HelpActionTitle",
```
Comment (ChrisPJohn) [inline SolastaCommunityExpansion/Diagnostics/OfficialBlueprints/ConditionDefinition/ConditionTrueStrike.json:253]: This looks like an accidental change of the TA definition?

**Review by ChrisPJohn — COMMENTED** [review]
A couple things I noticed when skimming through. The big one being it looks like the help action changes the text of the true strike condition so we might have another place we need to make sure we use a copy.

**Review by ChrisPJohn — APPROVED** [review]

File: SolastaCommunityExpansion/Diagnostics/CommunityExpansionBlueprints/ItemDefinition/Enchanted_Quarterstaff_Acuteness.json (hunk 1 above)
Comment (PhilPJL) [reply SolastaCommunityExpansion/Diagnostics/CommunityExpansionBlueprints/ItemDefinition/Enchanted_Quarterstaff_Acuteness.json:382]: previous export was exporting changes made by CE

File: SolastaCommunityExpansion/Diagnostics/OfficialBlueprints/ConditionDefinition/ConditionTrueStrike.json (hunk 2 above)
Comment (PhilPJL) [reply SolastaCommunityExpansion/Diagnostics/OfficialBlueprints/ConditionDefinition/ConditionTrueStrike.json:253]: Yes 

```
        private static void LoadHelpPower()
        {
            var effectDescription = new EffectDescription();

            effectDescription.Copy(TrueStrike.EffectDescription);
            effectDescription.SetRangeType(RuleDefinitions.RangeType.Touch);
            effectDescription.SetDurationType(RuleDefinitions.DurationType.Round);
            effectDescription.SetTargetType(RuleDefinitions.TargetType.Individuals);
            effectDescription.EffectForms[0].ConditionForm.ConditionDefinition.GuiPresentation.SetDescription("Condition/&HelpActionDescription");
            effectDescription.EffectForms[0].ConditionForm.ConditionDefinition.GuiPresentation.SetTitle("Condition/&HelpActionTitle");
```

**Pull request merged**